SETUP.md

```diff
@@ -0,0 +1,101 @@
+# Setup Instructions
```

Comment:
I think this section document can be omitted, and instead send instructions to install PuTTY and familiarize with terminal commands and SSH.